library/idr/externalDBs.py

```diff
@@ -0,0 +1,81 @@
+"""
+Helper functions for accessing the data from other databases from within IPython notebooks.
+"""
+import requests
+import pandas
+from io import StringIO
+from pandas import DataFrame
+from pandas import read_csv
+from pandas import concat
+import numpy as np
+
+def genes_of_interest_fromGO(goTerm):
```

Comment:
I'm unsure about this capitalization. Refer to PEP8 and run this throw a linter/flaker.